lib/pure/asyncdispatch.nim

```diff
@@ -9,7 +9,7 @@
 
 include "system/inclrtl"
 
-import os, oids, tables, strutils, times, heapqueue
+import os, oids, tables, strutils, times, heapqueue, queues
```

Comment:
If you count `heapqueue`, then there 3 queue modules:
`heapqueue`, `dequeue`, and deprecated `queues`.